prometheus/config.go

```diff
@@ -31,12 +31,18 @@ func GetScrapeConfig(scrapes map[string]Scrape) string {
 	}
 	templateString := `
 scrape_configs:{{range .}}
+{{- if .ScrapeType}}
+  - job_name: "{{.ServiceName}}"
```

Comment:
Maybe it makes sense to move this line above the `if` statement since it is applied always no matter of the condition.